src/main/java/se/sundsvall/seabloader/api/InvoicesResource.java

```diff
@@ -40,13 +40,14 @@ class InvoicesResource {
 
 	@PostMapping(consumes = {
 		APPLICATION_OCTET_STREAM_VALUE, APPLICATION_XML_VALUE
```

Comment:
Can this really return APPLICATION_XML_VALUE?

Reply:
"consumes" corresponds to the request header "Content-Type".

It has nothing to do with the any return values.

Reply:
Sorry. I was not reading the `consumes` part. My mind autofilled `produces`.